File: app/src/main/java/com/example/myapplication/fragments/manga_fragments/manga_new_releases_mvp/MangaNewReleaseInterface.java
package com.example.myapplication.fragments.manga_fragments.manga_new_releases_mvp;

import com.example.myapplication.models.mangamodels.DiscoverMangaModel;

import java.util.List;
import java.util.Map;

public interface MangaNewReleaseInterface {
    void onGetNewReleasesDataSuccess(List<DiscoverMangaModel> mangaNewReleaseResultModels, String hitStatus, Map<String, String> cookies);

    void onGetNewReleasesDataFailed();
}

File: app/src/main/java/com/example/myapplication/fragments/manga_fragments/manga_new_releases_mvp/MangaNewReleaseInterface.kt
package com.example.myapplication.fragments.manga_fragments.manga_new_releases_mvp

import com.example.myapplication.models.mangamodels.MangaNewReleaseResultModel

interface MangaNewReleaseInterface {
    fun onGetNewReleasesDataSuccess(mangaNewReleaseResultModels: List<MangaNewReleaseResultModel>, hitStatus: String)

    fun onGetNewReleasesDataFailed()
}
